apps/web/components/eventtype/RecurringEventController.tsx

```diff
@@ -47,8 +47,8 @@ export default function RecurringEventController({
           <Alert severity="warning" title={t("warning_payment_recurring_event")} />
         ) : (
           <>
-            <div className="relative flex items-start">
-              <div className="flex h-5 items-center">
+            <div className="absolute flex items-start">
```

Comment:
Is the absolute needed? Shouldn't flex-start suffice?

Reply:
agree. i dont think absolute should be used here. it would be better to stay flex but in the same container so it doesn't move down

Reply:
oops yes! removing wasn't meant to be on the commit and from previous testing